### DISCUSSION TITLE
Sofar solax - able to read but not write

### DISCUSSION THREAD

**Discussion by josefkrtil** [wills106/homeassistant-solax-modbus#645] [Q&A]
I have Sofar solax connected via modbus to HA. I can read all, but I am not able to change anything. On wiki is not documentation about Sofar solax. Will you help me, please?

**Comment by wills106** [comment]
Do you have a Sofar or a SolaX Inverter? They are two different manufactures.
Which plugin are you using when you setup the Integration?

**Reply by josefkrtil** [reply]
I have Sofar and I am using sofar plugin.

**Comment by DennyRuza** [comment]
Hi Josef,

I have just the opposite issue - I can control the inverter, however, I do not see any sensor values.

My setup uses Waveshare RS485 to USB connected using the following manual:
https://homeassistant-solax-modbus.readthedocs.io/en/latest/sofar-installation/

In HA, the inverter is set up with the "Serial" interaface and the "sofar" Inverter type, and other default values.

I did not set up anything directly on the Inverter.
